docker-compose.platform.yml

```diff
@@ -129,5 +129,6 @@ services:
       - ./configs/${NETWORK:?err}/dapi/nginx/errors.grpc_conf:/etc/nginx/conf.d/errors.grpc_conf
 
 volumes:
-  drive_mongodb:
-  drive_leveldb:
+  drive_mongodb_data:
+  drive_leveldb_data:
```

Comment:
It's more like `drive_abci_leveldb`. Isn't it? So we are prefixing with a service name.